lib/perl/Genome/Test/Command/RunPteroIntegrationTests.pm

```diff
@@ -2,14 +2,18 @@ package Genome::Test::Command::RunPteroIntegrationTests;
 
 use strict;
 use warnings FATAL => 'all';
-use Genome;
+use above "Genome";
```

Comment:
It seems like the caller should be the thing that does `use above`.  By the time we reach this `use above`, we're already using the version of this module contained in `PERL5LIB` and not the version in the desired checkout.